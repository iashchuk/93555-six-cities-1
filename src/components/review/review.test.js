import React from "react";
import renderer from "react-test-renderer";
import Review from "./review.jsx";

const user = {
  name: `Zak`,
  avatarUrl: `img/avatar.src`
};
const rating = 80;
const comment = `Bathed in the nature. Completely unplugged. Unforgettable`;
const date = `1995-12-17T03:24:00`;

describe(`Review`, () => {
  it(`renders correctly`, () => {
    const component = renderer
      .create(
          <Review user={user} rating={rating} comment={comment} date={date} />
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
