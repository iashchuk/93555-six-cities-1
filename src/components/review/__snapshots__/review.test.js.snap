// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar.src"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Zak
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "1600%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Bathed in the nature. Completely unplugged. Unforgettable
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      December 1995
    </time>
  </div>
</li>
`;
